doc/releases/tentacle.rst

```diff
@@ -0,0 +1,395 @@
+========
+Tentacle
+========
+
+Tentacle is the 20th stable release of Ceph.
+
+v20.2.0 Tentacle
+================
+
+
+Highlights
+----------
+
+*See the sections below for more details on these items.*
+
+RADOS
+
+* FastEC: Long expected performance optimizations were added for EC pools.
+* BlueStore: Users can expect to see improved compression and a new,
+  faster WAL (write-ahead-log).
+* Data Availability Score: Users can now track a data availability score
+  for each pool in their cluster.
+
+Dashboard
+
+* Support has been added for NVMe/TCP (gateway groups, multiple
+  namespaces), multi-cluster management, oAuth2 integration, and enhanced
+  RGW/SMB features including multi-site automation, tiering, policies,
+  lifecycles, notifications, and granular replication.
+
+RBD
+
+* New live migration features: RBD images can now be instantly imported
+  from another Ceph cluster (native format) or from a wide variety of
+  external sources/formats.
+* There is now support for RBD namespace remapping while mirroring between
+  Ceph clusters.
+* Several commands related to group and snap info were added or improved,
+  and `rbd device map` now defaults to msgr2.
+
+MGR
+
+* Users now have the ability to force-disable always-on modules.
+* The restful and zabbix modules (deprecated since 2020) have been
+  officially removed.
+
+RGW
+
+* Lua scripts will not run against health checks, properly quoted ETag
+  values returned by S3 CopyPart, PostObject and CompleteMultipartUpload
+  responses.
+* IAM policy evaluation now supports conditions ArnEquals and ArnLike,
+  along with their Not and IfExists variants.
+
+CephFS
+
+* Directories may now be configured with case-insensitive or normalized
+  directory entry names.
+* Modifying the FS setting variable "max_mds" when a cluster is unhealthy
```

Comment:
Suggest double backticks instead of ", and around the --yes-i-really-mean-it literal